QRSDetectorOffline.py

```diff
@@ -115,15 +116,13 @@ def __init__(self, ecg_data_path, verbose=True, log_data=False, plot_data=False,
 
         if verbose:
             self.print_detection_data()
-
+        filename = ecg_data_path.split("/")[1].split(".")[0]
```

Comment:
Again good point that files created like that will not have corresponding timestamp based names (there could occur difference in seconds value). But keeping timestamp in file names is - instead of just file path as you suggested - is important for sake of keeping logs and plots from multiple execution without overwriting any files from previous experiments. That is why the only change we suggest would be something like this:

	**log_timestamp = strftime("%Y_%m_%d_%H_%M_%S", gmtime())**
        if log_data:
            self.log_path = "{:s}QRS_offline_detector_log_{:s}.csv".format(LOG_DIR, **log_timestamp**)
            self.log_detection_data()

        if plot_data:
            self.plot_path = "{:s}QRS_offline_detector_plot_{:s}.png".format(PLOT_DIR, **log_timestamp**)
            self.plot_detection_data(show_plot=show_plot)

Now the filenames will correspond to each other with timestamp and we will have unique filenames.

Again as mentioned in PR comment - we cannot accept these changes right now but will remember about these suggestions in the future.